cmd/zrepl/mgmt_conn.c

```diff
@@ -238,40 +239,14 @@ scan_conn_list(void)
 	return (rc);
 }
 
-/*
- * Try to obtain controller address from zfs property.
- */
-static int
-get_controller_ip(objset_t *os, char *buf, int len)
```

Comment:
I remember I had to pull a few zfs header files because of this function to mgmt_conn.c. Most likely dmu* and dsl* header files are not needed anymore. Would be good if we can clean them up from this file as part of this change.

Reply:
makes sense.. will do that